python/openinference-semantic-conventions/src/openinference/semconv/trace/__init__.py

```diff
@@ -301,3 +309,20 @@ class OpenInferenceSpanKindValues(Enum):
 class OpenInferenceMimeTypeValues(Enum):
     TEXT = "text/plain"
     JSON = "application/json"
+
+
+class OpenInferenceLLMSystemValues(Enum):
+    OPENAI = "openai"
+    ANTHROPIC = "anthropic"
+    COHERE = "cohere"
+    MISTRALAI = "mistralai"
+
+
+class OpenInferenceLLMProviderValues(Enum):
+    OPENAI = "openai"
+    ANTHROPIC = "anthropic"
+    COHERE = "cohere"
+    MISTRALAI = "mistralai"
+    GOOGLE = "google"
+    azure = "azure"
+    aws = "aws"
```

Comment:
caps?

Reply:
also mismatched with js, js has gcp instead of google

Reply:
this is lowercase in genai so i'm leaving it lowercase.

Reply:
Oh you meant the key. Thank you